analyses/molecular-subtyping-LGAT/04-LGAT-compile-subtypes.Rmd

```diff
@@ -296,6 +316,14 @@ all_subtype <- bind_rows(all_rna_subtype,
                                   is.na(Kids_First_Biospecimen_ID_RNA))  ~ "LGG, To be classified",
     # when molecular_subtype is not NA 
     TRUE ~ stringr::str_c("LGG, ", molecular_subtype))) %>%
+  # recode if glialneuronal tumors
+  dplyr::mutate(molecular_subtype = case_when(str_detect(pathology_free_text_diagnosis,
```

Comment:
What about cases where the `pathology_diagnosis == "Ganglioglioma"`? Do we want to recode those as well?  I would expect so based on #1086

Reply:
Yes - per #1074, the Gangliogliomas should be annotated as `Ganglioglioma, subtype` instead of LGG or GNT @kgaonkar6